editor/editor_system.cpp

```diff
@@ -223,6 +223,10 @@ void EditorSystem::drawDefaultUI(float deltaMilliseconds)
 
 	static String loadingScene;
 
+#ifdef RELEASE_BUILD
```

Comment:
Use `#if !(defined(DEBUG) || defined(_DEBUG))` instead. The `RELEASE_BUILD` macro should be dropped.